examples/models/llava/export_llava.py

```diff
@@ -137,10 +131,21 @@ def forward(self, images):
         dynamic_shapes=dynamic_shapes,
     ).capture_pre_autograd_graph()
 
+    # quantizer
+    quantizer = XNNPACKQuantizer()
+    quantizer.set_global(get_symmetric_quantization_config())
+    prepared = prepare_pt2e(manager.pre_autograd_graph_module, quantizer)
+
+    # calibrate once
+    prepared(*manager.example_inputs)
+
+    # convert quantized
+    converted = convert_pt2e(prepared)
+
```

Comment:
This is part of the API in `LLMEdgeManager`: https://fburl.com/vnyea8af. Can you use that API?